src/Chapter1.hs

```diff
@@ -631,9 +642,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Integral a => a -> a
+sumLast2 n = a + b
+    where
+        a = (abs n) `mod` 10
+        b = ((abs n) `div` 10) `mod` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
abs n `mod` 10
abs n `div` 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(a, x) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Nice! Thank you.